db/migrate/20150112134701_create_rounds.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRounds < ActiveRecord::Migration
```

Comment:
@songjiayang OK then, this PR is ready to merge. Merge it if you think it's ready too.